tsl/test/shared/sql/constify_timestamptz_op_interval.sql

```diff
@@ -0,0 +1,11 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+
+-- all of these should have all chunk exclusion happening at plan time
+EXPLAIN SELECT time FROM metrics WHERE time < '2000-01-01'::timestamptz - '6h'::interval;
```

Comment:
Maybe use `EXPLAIN (VERBOSE ON) SELECT ...`. Could be good for debugging to see the expression that it is being re-written to.

Reply:
this wouldnt show more than whats already shown but tests are still WIP

Reply:
VERBOSE shows target lists and none of this PR modifies target lists, this is an internal constification to make those expressions available for plan time chunk exclusion, you can only see the effect in chunkappend execution time exclusion not having to do anything